cli_tools/gce_ovf_import/ovf_importer/ovf_importer.go

```diff
@@ -132,11 +132,15 @@ func (oi *OVFImporter) buildDaisyVars(
 	if bootDiskGcsPath != "" {
 		varMap["boot_disk_file"] = bootDiskGcsPath
 	}
-	if oi.params.Network != "" {
-		varMap["network"] = fmt.Sprintf("global/networks/%v", oi.params.Network)
-	}
 	if oi.params.Subnet != "" {
-		varMap["subnet"] = fmt.Sprintf("regions/%v/subnetworks/%v", region, oi.params.Subnet)
+		varMap["subnet"] = param.GetRegionalResourcePath(region, "subnetworks", oi.params.Subnet)
+		// When subnet is set, we need to grant a value to network to avoid fallback to default
+		if oi.params.Network == "" {
+			varMap["network"] = ""
+		}
```

Comment:
Why is this needed? To make sure there's a "network" variable with an empty string value? Why?

Reply:
This logic exists in importer and exporter - since subnet can belong to a user-defined network, we don't want network value fallback to "default". An empty 'network' can prevent that from happening.

Please let me know if ovf importer doesn't need that for some reason.